### ISSUE TITLE
Bugs

### ISSUE THREAD

**Issue by Fasal-Official** [csoc-de/Shifts#11]
Hello @fkirchesch 

I was using your app on my development NextCloud instance to make sure it reaches my expectations and requirements. As far as I can see, this seems matching my requirement however, there are some bugs I've noticed while going through the App. I've listed them below for your reference:

1. Require English language support (not sure why you didn't put this by default to make it useful all over the world)
2. In Settings > Shifts under Administration tab - There is no message saying successfully saved when I hit "Save on computer". Also the documentation about this section is still missing some explanations, so better you edit it.
3. In the App interface, 'Shift types' section: when creating a new type, the 'Stop time' clock is not showing bottom columns unless I zoom out in Chrome browser. 
![Shift type clock display](https://user-images.githubusercontent.com/46318630/143684687-e2e14222-7b95-43ca-8997-c10f26cf8f1e.png)

If I zoom out the interface, it shows correctly and no way to scroll that box to get bottom clock numbers with 100% zoom.

4. In the above same interface, if I once click Weekly option and unselect it back then the field orders are going weird.
5. Once a shift type, and going to edit it then there is an option named 'rules' at the bottom and I don't get what that actually refers or why this option is. No information found in documentations.
6. In Shift schedule section, when I initially select Month view it displays the columns in a weird way. I've to refresh the page once and then select Month view again to get it correct. 
7. While assigning a new shift, only Mondays are selectably on the calendar (rest are in grey status). Image below:

![shift assign error](https://user-images.githubusercontent.com/46318630/143687454-f932c87a-0d7d-4d34-ac2b-eb46f8c7f80b.png)

In above image. the red section is not selectable and only Mondays available to select. Why is this occurring? 

Please have a review of the above, and sort the bugs. Also, include the English lang support and make the documentation more clear. 

Thanks,
Fasal

**Comment by fkirchesch** [comment]
Furthermore, i cant reproduce your bug listed as number 3. May i ask what resolution you use on your monitor ?

**Comment by fasalsh** [comment]
That is on shift type section, when trying to create a new type the stop time clock doesn't show bottom rows. If I zoom out my Chrome window, then it shows fine. I'm not in front of my system, so I can share a video of it tomorrow to understand it better.

Are you in Linkedin? If so, share me your id so we could connect.

**Comment by fkirchesch** [comment]
Im aware of the problem you have, but i cannot reproduce it at mine, im using a normal full hd 1920x1080 Monitor and i have since tested it on other monitors with different resolutions and then i can reproduce similar sizing issues. 

I dont have LinkedIn, but i think we can use this issue to communicate.

**Comment by fasalsh** [comment]
Here is video representation of the issues described:

https://user-images.githubusercontent.com/46318630/144060630-b8b8bc0e-5b15-4438-b0b7-57d9dad66f58.mp4

**Comment by fasalsh** [comment]
Any thoughts? @fkirchesch

**Comment by fkirchesch** [comment]
Oh hey,
sorry, thought i had already wrote a reply.
I have since been able to reconstruct or reproduce this behaviour and pinned it down to the resolution of the used monitor. I will try to fix it by centering every dialog to the center of the screen. This fix will be in the new version which will hopefully come out next week.
Best regards 

Fabian

**Comment by fasalsh** [comment]
cool, thanks @fkirchesch
Waiting for a release on this app with all these bug fixes 👍

**Comment by fkirchesch** [comment]
Hey @fasalsh, 
its me again :D.
I have just released a new Version which should fix most of the bugs you have found.
I unfortunatly could not get the buggy Dialog to appear in the center of the screen. I have realigned the parent Inputs which should make it easier to access theses dialogs, but it wont eliminate the issue completely. I have not found a way to resolve this.

I hope you still like it :D

Best regards 

Fabian
